L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -403,6 +401,17 @@ void L1TGlobalProducer::produce(edm::Event& iEvent, const edm::EventSetup& evSet
       m_l1GtMenu->print(std::cout, printV);
 
     m_l1GtMenuCacheID = l1GtMenuCacheID;
+
+    //for getting model version to condition class via GlobalBoard.runGTL, comes from menu rather than config
+    if (gtParser.gtTriggerMenuName() == "L1Menu_Collisions2024_v1_0_0") {
+      m_AXOL1TLModelVersion = "";
```

Comment:
Wouldn't you simply put ```"GTADModel_v3"``` here for v1_0_0?

Reply:
fair point, I'll add a push and fix this